lint.sh

```diff
@@ -42,3 +42,6 @@ flake8 --show-source ${LINT_FILES}
 
 # Indents are four spaces, binary ops can start a line, and indent switch cases
 shfmt -i 4 -bn -ci -d .
+
+# Checking minimum python version
+vermin -t=3.6 --violations --eval-annotations ./pwndbg/
```

Comment:
Why do we need `--eval-annotations`?

Reply:
If that flag isn't necessary it can be removed. The flag had no effect and was put in if required by some of the code.

Reply:
I've gone ahead and removed the flag.